main.py

```diff
@@ -2,6 +2,52 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genre_names = ["Western", "Action", "Dramma"]
+    actors = [("George", "Klooney"),
+              ("Kianu", "Reaves"),
+              ("Scarlett", "Keegan"),
+              ("Will", "Smith"),
+              ("Jaden", "Smith"),
+              ("Scarlett", "Johansson")]
+
+    for genre in genre_names:
+        Genre.objects.create(
+            name=genre,
+        )
+
+    for first_name_, last_name_ in actors:
+        Actor.objects.create(
+            first_name=first_name_,
+            last_name=last_name_,
+        )
+
+    Genre.objects.filter(
+        name="Dramma",
+    ).update(name="Drama")
+
+    Actor.objects.filter(
+        last_name="Klooney",
+    ).update(last_name="Clooney")
```

Comment:
The condition says you have to change George Klooney's surname, not all Klooney surnames.

Reply:
Good tought - fixed.